ramp-frontend/ramp_frontend/templates/teaching.html

```diff
@@ -27,27 +27,107 @@ <h4 class="modal-title" id="myModalLabel">{{ category }}</h4>
+                RAMP challenges begin with an interesting supervised prediction
+                problem for which there is data. Domain experts then work with
+                experienced data scientists to clean and curate the
+                data and formalize the problem. This process can take two weeks to
+                six months, and results in a 'starting kit', which typically
+                consists of:
+
+                <ul>
+                    <li>
+                        A jupyter notebook that introduces the predictive
+                        problem, provides some background information, exploratory
+                        data analysis and data visualisation, explains the workflow
+                        and provides a simple example solution.
+                    </li>
+                    <li>
+                        A 'problem.py' file that parametrizes the setup and uses
+                        building blocks from <a href="https://github.com/paris-saclay-cds/ramp-workflow">
+                        RAMP workflow</a>.
+                    </li>
+                    <li>
+                        The data, split into training and test subsets.
+                    </li>
+                    <li>
+                        A requirements file detailing all the required software
+                        packages.
+                    </li>
+                </ul>
+            </p>
+            <p>
+                A challenge event can now be organised. These events may
+                be part of a post-graduate level programme or a 'hackathon'.
+                If the data science problem requires the mastering
+                of a specific tool, the RAMP event can be preceded by a
+                Training Sprint for explaining specific tools to the participants.
+            </p>
+
+            <p>
+                RAMP data challenges has been used for teaching in many courses
+                at various levels. Students usually love the platform for:
+                <ul>
+                    <li>its competitive gaming aspect,</li>
+                    <li>being able to learn from each other in the open phase, and</li>
+                    <li>to compete as a group with other student groups on the same problem.</li>
+                </ul>
+            </p>
+
+            <br>
+
+            <p>
+                <h1 style='font-weight: bold;'>Using RAMP for teaching</h1>
+                If you are interested in using the platform for teaching,
+                We can open a new event for any <a href="/problems">
+                RAMP challenge</a>. You will have full control over the users
+                who can sign up for your event and the timeline the event
+                (start, moving from closed to open phase and end).
+            </p>
+
+            <p>
+                At the end of your challenge we will give you a csv file with all
+                the performance metrics of each student.
```

Comment:
it should be able to download this from the admin page from the event. It's very manual so far

Reply:
I don't see this option. Should it be under 'dashboard admin' or 'update event'? @glemaitre @maikia do you know if this option exists?

Reply:
yes it is not available yet but if we say this it should be added